.rubocop_todo.yml

```diff
@@ -17,6 +17,12 @@ Metrics/BlockLength:
 Metrics/ClassLength:
   Max: 117
 
+# Offense count: 1
+# Configuration parameters: IgnoredMethods.
+Metrics/CyclomaticComplexity:
+  Exclude:
+    - 'lib/slack/block_kit/element/multi_static_select.rb'
```

Comment:
Excluded this file due to the new `.map(&:as_json)` call. I opted to ignore this in Rubocop, as extracting it would make the line look out of place.

```rb
            action_id: @action_id,
            options: @options&.map(&:as_json),
            option_groups: @option_groups&.map(&:as_json),
            initial_options: @initial_options&.map(&:as_json), # new
            confirm: @confirm&.as_json,
            max_selected_items: @max_selected_items
```